_test/test_mpi_wrappers/test_cluster_wrapper.m

```diff
@@ -158,6 +158,18 @@ function test_check_progress_disp_results(obj)
             assertEqual(cluster.log_value,ref_string);
             
         end
+        
+        function test_parpool_uses_logical_cores_if_requested(obj)
+            [physical_cores, logical_cores] = get_num_cores();
+            if physical_cores >= logical_cores
```

Comment:
If `logical_cores == physical_cores` then the ` clust.init(physical_cores + 1, ...)` call below will fail because not enough cores will be available.

The `>` case won't happen, so that could be taken out. I think I just put it in because the rest of the test is not valid for that case, although it's probably better we just let the test fail, because something would obviously be very wrong.

Reply:
I would take the `>` out for clarity.